_data/accessibility-tests/banner.yml

```diff
@@ -0,0 +1,121 @@
+title: Banner
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey meaning in the banner.
+    summary_additional: When viewing the banner, you never have to rely on color alone to understand meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Text meets color contrast requirements.
+    summary_additional: |
+      When you open the banner and use ANDI or another color contrast analyzer to look at the hex numbers,
+      the contrast between the banner text and background color is at least 4.5:1.
```

Comment:
Curious, for components with settings to customize colors, should we add a note that they'll need to test themselves if they customize colors? We do offer color contrast checks that will warn them if the colors aren't compliant so maybe we don't need to worry about noting here.

What do you think?

Reply:
I don't think it is necessary to add an additional note here. We explicitly state at the top of the page that each of these tests should be completed after being implemented in a project ("USWDS tests components in isolation. You need to test the banner component in the context of your own site to ensure compliance with Section 508 accessibility standards."), so users should be testing regardless. Any objections to moving forward as-is?

Reply:
I'm leaning more with, Amy. I think we can keep it as is (but I also see your POV as well, Charlie).